crypto/asn1/a_sign.c

```diff
@@ -198,10 +204,16 @@ int ASN1_item_sign_ctx(const ASN1_ITEM *it,
 
     }
 
-    inl = ASN1_item_i2d(asn, &buf_in, it);
+    buf_len = ASN1_item_i2d(asn, &buf_in, it);
+    if (buf_len <= 0) {
+        outl = 0;
```

Comment:
Should this be `inl = 0` here?

Reply:
That's not necessary right now, as it's already initialized to 0, and is not changed in the code above.

Reply:
Ah! Right. Thanks for the clarification.